misc/ls-extensions/modules/sa-modeling-service/src/main/java/io/ballerina/componentmodel/servicemodel/ServiceModelGenerator.java

```diff
@@ -0,0 +1,45 @@
+package io.ballerina.componentmodel.servicemodel;
```

Comment:
Missing license header

Reply:
Please add license headers for all the other newly added files as well

Reply:
Added